pkg/mls/store/queries.sql

```diff
@@ -225,3 +225,22 @@ INSERT INTO inboxes(id)
 ON CONFLICT (id)
 	DO UPDATE SET
 		updated_at = NOW();
+
+-- name: GetOldWelcomeMessages :one
+SELECT COUNT(*)::bigint as old_message_count
+FROM welcome_messages
+WHERE created_at < NOW() - make_interval(days := @age_days);
```

Comment:
Given the sheer amount of welcome messages, could this query timeout or block with a big enough @age_days?

Reply:
yes. I expect this to be slow. I will watch the logs and see if it needs changing.